pyomo/contrib/pynumero/algorithms/solvers/tests/test_scipy_solvers.py

```diff
@@ -16,6 +16,12 @@
 if not scipy_available:
     raise unittest.SkipTest("SciPy is needed to test the SciPy solvers")
 
+from pyomo.contrib.pynumero.asl import AmplInterface
+if not AmplInterface.available():
+    raise unittest.SkipTest(
+        "Pynumero needs the ASL extension to run SciPy solver tests"
+    )
```

Comment:
Isn't this redundant?  That is, don't we need either this block of code or the `@skipUnless` guards on the class or test methods -- but not both?

Reply:
Yep, don't know why I did both